SimPEG/NSEM/Utils/dataUtils.py

```diff
@@ -56,11 +56,11 @@ def skindepth(rho, freq):
 def rec2ndarr(x, dt=float):
     return x.view((dt, len(x.dtype.names)))
 
-def makeAnalyticSolution(mesh, model, elev, freqs):
-    from SimPEG import MT
+def makeAnalyticSolution(mesh,model,elev,freqs):
```

Comment:
what type of analytic solution is this? halfspace, layered earth? I would suggest including the assumption in the function name